cmiles/generator.py

```diff
@@ -3,6 +3,7 @@
 
 """
 import warnings
+import collections
 from copy import deepcopy
 import cmiles as c
 
```

Comment:
Is this the only file you import openeye/rdkit?

Reply:
I also import it in utils.